lua/autorun/client/TrueCulling.lua

```diff
@@ -0,0 +1,68 @@
+jit.on()
+
+CULLFRONT = math.sin(0.57357644) -- Degrees forward angle in sine
```

Comment:
Is this supposed to be related to the field of view? FOV is not static, and if you want to do this properly, it's not that hard to check the actual frustum instead of bounding it with a cone

Reply:
I have some code somewhere that will get the actual view frustum after other scripts have modified it, I can dig it out for you if you want to try to fix this